packages/docs/docs/contributing/bounty.md

```diff
@@ -0,0 +1,35 @@
+---
+image: /generated/articles-docs-contributing-bounty.png
+title: How to take a bounty issue
+sidebar_label: How to take a bounty issue
+crumb: Contributing
+---
+
+It’s nice to see community members contribute to Remotion’s code base. There are rewards for specific issues on GitHub to keep engagement high. These rewards are called bounties.
+
+## Find bounties
+
+All bounties can be tracked in Remotion’s GitHub repository under issues. Issues with a bounty have a label (💎 Bounty) on them. You can filter issues based on this bounty label to get an overview of all <a href="https://github.com/remotion-dev/remotion/issues?q=is%3Aopen+label%3A%22%F0%9F%92%8E+Bounty%22+sort%3Aupdated-desc">open</a> or <a href="https://github.com/remotion-dev/remotion/issues?q=label%3A%22%F0%9F%92%8E+Bounty%22+sort%3Aupdated-desc+is%3Aclosed">closed</a> issues. Furthermore, bounties are announced in the <a href="https://remotion.dev/discord">Remotion Discord Server</a>.
+
+## Taking a bounty
+
+Initially, there is no difference to a common contribution, that has no bounty. To tackle an issue, you simply need to comment with the intention of solving that issue. For issues with bounties, the Remotion team does a quick screening to see if the experience matches the problem difficulty. In the best case, describe your approach of solving that issue when initially commenting on it. If everything looks good, you get the issue assigned. The assignment is done on a first-come-first-served basis, and only one bounty at a time is assigned to the same contributor.
+
+## Before starting to solve the issue
+
+Bounties are paid through a third-party platform (i.e., Algora). Before you start solving the issue, it’s recommended to create an account on <a href="https://algora.io/">Algora</a>, where you also need to set up and connect your Stripe account. Once you have been assigned the issue and created an account on Algora, you are ready to go. You can start hacking.
+
+## Start with hacking
+
+It's assumed that you start working on the issue right after it has been assigned to you. Getting stuck is not an issue, but you should communicate it. By telling and describing the challenges, you might get some input from the community, which can help you to solve the issue faster. If there is no communication done within one week since you started hacking, it is assumed you left the issue, and the bounty is going to be reassigned to someone else.
+
+## Help during hacking
+
+During Hacktoberfest (i.e., October), the Remotion team provides guidance to some degree, since there is a focus on new contributors. Besides Hacktoberfest, no guidance is provided when tackling a bounty issue. Of course, questions about stylistic matters and important implementation behaviors are welcome.
```

Comment:
Sounds a bit too strict for me, there is definitely guidance.

The difference is that during October, beginners are welcome while the rest of the year, people who have an idea on how to solve the issue should claim it (which might still need guidance though)